pagopa/io-app
# feat: [IOPID-742] New email validation screens added

## Short description
In this PR, the screen allowing the user to validate the e-mail was created ([figma here](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=2522%3A23388&mode=dev)). The same screen, if the email is validated, shows a success screen ([figma here](https://www.figma.com/file/BEXvjQmBkMRRFiNcMgE8g3/Login-Veloce?node-id=2522%3A23897&mode=dev))

## List of changes proposed in this pull request
- Screen allowing the user to verify the email  
- screen notifying the user that the email has been correctly verified 
- integrate new text
- add feature flag CDU_NEW_FLOW
- add tests on NewRemindEmailValidationOverlay
- delete DE texts 

## How to test
This screen is to be integrated later, so for now for testing purposes it must be displayed by 'forcing' it.
Into the io-app, in TabNavigator.tsx file change line 82 from MessagesHomeScreen to NewRemindEmailValidationOverlay.
To display the screen allowing the user to verify the email, on line 41 the const isEmailValidated must be set to false. To see the validated email screen the same value must be set to true. 
In the screen that allows the user to verify the email upon clicking on the button at the bottom of the page, a timeout of 10 seconds is triggered which disables the button and a call is made which sends the email to the user for validation. 
In the success screen upon clicking on the footer button the user continues the flow.
